lib/pages/scout/dialogs/RedeemDialog.dart

```diff
@@ -388,11 +398,13 @@ class _RedeemDialogState extends State<RedeemDialog> {
                       child: TextButton(
                         onPressed: () {
                           lspController.redeem();
-                          Navigator.pop(context);
-                          showDialog(
-                              context: context,
-                              builder: (BuildContext context) =>
-                                  confirmTransaction(context, true, ""));
+                          //check if there was a success in redeeming then show dialog
+                          if (show! == true) {
+                            showDialog(
+                                context: context,
+                                builder: (BuildContext context) =>
+                                    confirmTransaction(context, true, ""));
+                          }
```

Comment:
If the user has rejected the transaction we will want a dialog to notify the user that the transaction has either failed/rejected

Reply:
There is no Ui for this